internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -703,6 +734,8 @@ func resourceKubernetesClusterNodePoolRead(d *pluginsdk.ResourceData, meta inter
 		d.Set("fips_enabled", props.EnableFIPS)
 		d.Set("ultra_ssd_enabled", props.EnableUltraSSD)
 		d.Set("kubelet_disk_type", string(props.KubeletDiskType))
+		d.Set("scale_down_mode", string(props.ScaleDownMode))
```

Comment:
This will need to be defaulted here because of the way the API behaves: 
```suggestion
	scaleDownMode := string(containerservice.ScaleDownModeDelete)
	if v := props.ScaleDownMode; v != "" {
		scaleDownMode = v
	}
	d.Set("scale_down_mode", scaleDownMode)
```